_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -53,6 +53,11 @@ <h3>Snap Support</h3>
 
 {{^advanced}}
 {{>installcertbot}}
+  <p>
```

Comment:
nit: I personally think it'd be a good idea to make this a separate numbered released instruction by enclosing it with `<li>` tags. I worry that without it, people might miss it.

Reply:
You are right. I put it in a dedicated step.